addons/website_slides/controllers/main.py

```diff
@@ -624,6 +624,13 @@ def slide_archive(self, slide_id):
 
         return False
 
+    @http.route('/slides/slide/preview', type='json', auth='user', website=True)
```

Comment:
Just to be more explicit I would maybe name the route to /slides/slides/toggle_is_preview to explicitely tell it changes the is_preview flag and has nothing to do with preview of a given slide.

Reply:
By the way maybe the selector and JS file should be named accordingly, like slides_slides_toggle_is_preview. That way everything is well named :) .